apps/shared/function.h

```diff
@@ -78,6 +78,7 @@ class Function : public ExpressionModelHandle {
     }
     bool isActive() const { return m_active; }
     void setActive(bool active) { m_active = active; }
+    void setMyColor(KDColor color) {m_color = color; }
```

Comment:
```suggestion
    void setMyColor(KDColor color) { m_color = color; }
```

Reply:
In fact, is it possible to give a better name to the function?

Reply:
setGraphColor ?